DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/os/linux/linux_sysctl_impl.py

```diff
@@ -0,0 +1,41 @@
+from dent_os_testbed.lib.os.linux.linux_sysctl import LinuxSysctl 
+class LinuxSysctlImpl(LinuxSysctl):
+    """
+        system kernel attributes manager
+    """
+    def format_get(self, command, *argv, **kwarg):
+        """
+        Get the attribute value
+        
+        """
+        params = kwarg["params"]
+        cmd = 'sysctl -n {} {}'.format(command, params["variable"])
```

Comment:
`sysctl` doesn't have a 'command'. `sysctl -n get ...` would be and invalid command.